src/Core/Style.js

```diff
@@ -138,10 +138,10 @@ function defineStyleProperty(style, category, parameter, userValue, defaultValue
         {
             enumerable: true,
             get: () => {
-                if (property) { return property; }
-                if (userValue) { return readExpression(userValue, style.context); }
+                if (property !== undefined) { return property; }
```

Comment:
you have to check for `null` too

Reply:
done. Using value != undefined.

Reply:
I got the information on internet https://codedamn.com/news/javascript/check-if-undefined-null and checked it with https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Inequality.